packages/editor-ui/src/components/Projects/ProjectCardBadge.vue

```diff
@@ -52,25 +52,28 @@ const projectState = computed(() => {
 	}
 	return ProjectState.Unknown;
 });
+
+const numberOfMembersInHomeTeamProject = computed(
+	() => props.resource.sharedWithProjects?.length ?? 0,
+);
```

Comment:
@cstuncsik please double check this is the correct source of truth

Reply:
to the extend of my knowledge.. we want to share how many users the resource is shared with, i'll ask in the ticket to double check

Reply:
That's the easier, the other one could only be problematic in the Home project because the resources are list from all available projects and you would need to get all the relations for all the projects

In a team project you just get it from the store (`projectsStore.currentProject.relations`)